### ISSUE TITLE
Create the ability to automate the update process. (Command line options? GUI?)

### ISSUE THREAD

**Issue by Truandale** [Recol/DLSS-Updater#23]
Hello. This is not a problem, this is a wish. Every day my PC wakes up at night, performs maintenance actions (updates programs, checks for viruses, etc.) and falls asleep.
Regarding dlss updater. If this is a console program, then it really lacks parameters for launching (specify the location of the paths where to look for games, for example). With the parameters, I would launch the bat file from the Task Scheduler, the program itself would update everything and close. 
Or indicate the same thing, but in the gui.
The program as an idea is simply great. It just seems to me that it lacks something to automate the process. Or I just didn't find it?

**Comment by Recol** [comment]
You are correct, this functionality doesn't exist, it is possible to implement, I'll consider possibly adding automation support in the future.

**Comment by rayman3003** [comment]
Good to have a GUI for this app

**Comment by Recol** [comment]
@rayman3003 There is work ongoing for this at https://github.com/Recol/DLSS-Updater/tree/ui-integration, it will take time however.

**Comment by rayman3003** [comment]
> @rayman3003 There is work ongoing for this at https://github.com/Recol/DLSS-Updater/tree/ui-integration, it will take time however.

Good to hear about it.

Desperately waiting for it.

**Comment by Recol** [comment]
@rayman3003 @rayman3003 Hi both, i set aside some time to build these, they're now available with version [2.5.0](https://github.com/Recol/DLSS-Updater/releases/tag/V2.5.0). 

Cheers,
Deco

**Comment by Recol** [comment]
> > @rayman3003 @rayman3003 Hi both, i set aside some time to build these, they're now available with version [2.5.0](https://github.com/Recol/DLSS-Updater/releases/tag/V2.5.0).
> > Cheers, Deco
> 
> I think it doesn't work for custom game folders.
> 
> I added a random game (angry birds) to the DLSS Updater, that it's dll (msvcr100.dll) is old. But the DLSS Updater says "No updates available. You have the latest version." But as u can see in the screenshot, the dll (msvcr100.dll) of the game is old!
> 
> ![2024-11-19 10_54_51-](https://private-user-images.githubusercontent.com/17022701/387542976-92d02c11-50b1-4038-9d67-881e8403089a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIwMDg2MzYsIm5iZiI6MTczMjAwODMzNiwicGF0aCI6Ii8xNzAyMjcwMS8zODc1NDI5NzYtOTJkMDJjMTEtNTBiMS00MDM4LTlkNjctODgxZTg0MDMwODlhLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE5VDA5MjUzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzZDNhY2VjM2EwNjBlMGY1OGUwYWY3ZjBhMDdiZjI3YzMyNjE2MzFkM2E4NTA2NjRlZTI2MjNjMTA4ZGZjMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.fyTDW-xLErHqlhKAp-eBGxKgCTfeo6CnmZbhSELodYA)
This DLL is not for DLSS - or is it affiliated, this is part of the MVC DLL by Microsoft, the names of the DLL files can be found `[here] for DLSS(https://github.com/Recol/DLSS-Updater/tree/main/latest_dll)`.

**Comment by rayman3003** [comment]
> This DLL is not for DLSS - neither is it affiliated with it, this is part of the MVC DLL by Microsoft, the names of the DLL files can be found here for DLSS: https://github.com/Recol/DLSS-Updater/tree/main/latest_dll
> 
> Angry Birds does not support DLSS.

OK. My bad.

I though DLSS updater could update all dlls of any game.

**Comment by Recol** [comment]
@Truandale FYI since this feature request there's been a significant overhaul with the inclusion of a GUI, so it's now technically possible to automate using Power Automate (or some other utility) which allows for clicking the button, so i don't think it's required to implement explicitly, will close the issue as a result cheers.
